cli/ballerina-packerina/spotbugs-exclude.xml

```diff
@@ -25,6 +25,14 @@
         <Class name="org.ballerinalang.packerina.model.BaloToml"/>
         <Bug pattern="NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE"/>
     </Match>
+    <Match>
+        <Class name="org.ballerinalang.packerina.cmd.InitCommand"/>
+        <Bug pattern="NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE"/>
+    </Match>
+    <Match>
```

Comment:
Should this be removed once the AddCommand is implemented? If so please add comment to #26148 referring the PR.

Reply:
Yes, it will be removed.